x-pack/test/fleet_api_integration/apis/agents/unenroll.ts

```diff
@@ -208,16 +208,6 @@ export default function (providerContext: FtrProviderContext) {
           batchSize: 2,
         })
         .expect(200);
-
-      expect(unenrolledBody).to.eql({
-        agent1: { success: true },
-        agent2: { success: true },
-        agent3: { success: true },
-        agent4: { success: true },
-      });
-
-      const { body } = await supertest.get(`/api/fleet/agents`);
-      expect(body.total).to.eql(0);
```

Comment:
Should are tests use the action status API to verify it was completed or scheduled somehow?

Reply:
I can add a test for the `revoke:false` case. For force unenroll, there is no document created in `.fleet-actions`, so the action status API is not going to return it.

Reply:
updated